src/styles/blocks/card.scss

```diff
@@ -0,0 +1,139 @@
+.catalog {
+  display: grid;
+  margin: 0;
+
+  &__content {
+    display: grid;
+    margin: 0 auto;
+    padding: 50px 40px;
+    gap: 46px 48px;
+    grid-template-columns: repeat(1, 200px);
+
+    @media (min-width: 488px) {
+      & {
+        grid-template-columns: repeat(2, 200px);
+      }
+    }
+
+    @media (min-width: 768px) {
+      & {
+        grid-template-columns: repeat(3, 200px);
+        padding-inline: 40px;
+      }
+    }
+
+    @media (min-width: 1024px) {
+      & {
+        grid-template-columns: repeat(4, 200px);
+      }
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px #F3F3F3 solid;
+  border-radius: 5px;
+  transition: all 300ms ease-in-out;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    flex-shrink: 0;
+    height: auto;
+    width: 160px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+    aspect-ratio: 160 / 134;
+  }
+
+  &__name {
+    margin: 0;
+    margin-bottom: 4px;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $main;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    line-height: 14px;
+    font-size: 10px;
+    font-weight: 400;
+    color: $secondary;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    line-height: 14px;
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    color: $main;
```

Comment:
you can remove this style `font-weight: 400;` cause it works by default

please delete this value in other places of your code too